app/views/users/show_follow.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide(:title, @title) %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <%= gravatar_for @user %>
+      <h1><%= @user.name %></h1>
+      <span><%= link_to "view my profile", @user %></span>
+      <span><b>posts:</b> <%= @user.posts.count %></span>
```

Comment:
- I18n
- em dùng css để định kiểu chữ in đậm nhé, ko dùng thẻ `<b>`
- `@user.posts.size`